server/trafficrules/src/main/java/com/coffeejug/trafficrules/SecurityConfig.java

```diff
@@ -0,0 +1,20 @@
+package com.coffeejug.trafficrules;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+@Configuration
+@EnableWebSecurity

```

Comment:
Do we need spring security?

Reply:
I think yes, in future.

Reply:
Let's follow the YAGNI principle, please. Adding something for the future use is considered a bad practice

Reply:
ok